.github/workflows/unit-test-pdf.yml

```diff
@@ -0,0 +1,115 @@
+name: Unit Tests for PDF features
```

Comment:
Might be better to include all unit tests (PDF and others) in the same file for better maintainability.

Reply:
This would result in a gigantic workflow file, which I kind of want to avoid

Reply:
:+1: